integration/images.go

```diff
@@ -218,8 +225,8 @@ func addServiceAccountFlags(flags []string, serviceAccount string) []string {
 	return flags
 }
 
-func (d *DockerFileBuilder) BuildDockerImage(imageRepo, dockerfilesPath, dockerfile, contextDir string) error {
-	fmt.Printf("Building image for Dockerfile %s\n", dockerfile)
+func (d *DockerFileBuilder) BuildDockerImage(t *testing.T, imageRepo, dockerfilesPath, dockerfile, contextDir string) error {
+	t.Logf("Building image for Dockerfile %s\n", dockerfile)
```

Comment:
Not `d.logf`?

Reply:
`d.logf` is the outer `t *testing.T`, where what we pass here is from the inner `t.Run`.  I started with `d.logf` and it wasn't partitioning the logs as well as it should have been.  I'd probably rather drop `d.logf` entirely than switch back, since this was the bigger deal IIRC.

Reply:
Ah got it. If there's a way to drop `d.logf` that'd be nice, or at least a comment describing how they're different.

Reply:
I looked at where it was still used and it was a place we could switch to a `t.Logf` so I removed most of that stuff in the latest commit and made the type private.